### DISCUSSION TITLE
Film Reviews

### DISCUSSION THREAD

**Discussion by JPH71** [CodyBerenson/PGMA-Modernized#80] [Ideas]
The following Agents, CDUniverse, GayHotMovies and the new nymMedia support Film Reviews as these are frequently referred to on their Film webpages....

The good old favourites like GEVI and AEBN and to a large extent GayDVDEmpire either don't support reviews or don't use them enough (GayDVD)...

As these Agents support Scene breakdowns, I was thinking it may be better if we took the scene breakdowns and place them in the reviews section, rather than trawling through the summaries.... The choice can be determined by setting a preference

As Gayhotmovies can have scene breakdowns and reviews - we can set the first review to be filled with the scene breakdown??

Next question - I have restricted the amount of reviews to 4 - would you prefer all the reviews to show or be allowed to set a preference of how many you would want... this means if you set to ZERO, no reviews would show...

Attached is an example of a movie with a review:
![falcon Screenshot 2021-03-11 211955](https://user-images.githubusercontent.com/43102249/110856488-9b38a300-82af-11eb-97c1-7f23f08826e0.png)

**Comment by j-ktz** [comment]
@JPH71 This is awesome. You never cease to amaze me. My vote would be for an option to set how many reviews (including zero). I'm also okay with the first spot being reserved for a scene breakdown if there is one. Can't wait!!!

**Comment by JPH71** [comment]
Damn, You were superfast!

I was just checking my mail to see if the discussion had activated....

So as it stands

1. For Agents that have Scene breakdowns, but no reviews (Note Scenes show
best if they are less than 300 continuous characters)

- we will use the review section for the scene breakdowns

- One review slide per scene - In GEVI they put a lot of info in each scene

- In AEBN and GayHotmovies - we just have the actors and the length of the
scene

2. For Agents with Reviews but no scenes

- use the review section as designed

- applies to nymMedia and CDUniverse

3. For Agents with Reviews and  Breakdows

- applies to GayHotMovies and GayDVD Empire

- 1st and possibly 2nd Review Slot - has the scene breakdown as they are
basic on these agents

- others Slot onwards have reviews as normal


So for option 2 - we will have a preference that allows reviews and how
many... 0....10

For option 1 and 3 - we have an option to keep the scene breakdowns with
the summary as it is now or use the review section, this will work in
conjunction with Option 2 preference..
If you set Number of reviews to zero - you will only see the scene
breakdowns otherwise you will get reviews after the scene breakdowns....

**Comment by joca879** [comment]
Another enhancement :)
The functionality of the reviews seems interesting.
I agree with the @j-ktz sugestions

**Comment by joshallen13** [comment]
I'm curious as to what displays with the scene breakdown in the review module. Particularly, what is shown as a replacement for the Starred meter? Does it just not show anything there?

**Comment by JPH71** [comment]
It will take the scene breakdown out of the summary
